lib/components/Upload.js

```diff
@@ -119,6 +137,32 @@ export default class Upload extends Component {
     }
   }
 
+  onMedtronicSerialNumberInputChange() {
+    const { refs } = this;
+    let serial_number = refs.medtronic_serial_number;
```

Comment:
I think I would prefer in this instance not to rely on grabbing the value from `refs` when it's being passed into the function through the event (via `event.target.value`). Unless there's something about the `ref` pattern here that I'm missing.